hub/util/keys.py

```diff
@@ -17,24 +15,24 @@
 
 def get_chunk_key(key: str, chunk_name: str, commit_id: str) -> str:
     if commit_id == FIRST_COMMIT_ID:
-        return posixpath.join(key, CHUNKS_FOLDER, f"{chunk_name}")
+        return "/".join([key, CHUNKS_FOLDER, f"{chunk_name}"])
```

Comment:
posix path is quite slow. These keys are only used by our storage providers so just this simple join should do the trick.

Reply:
Out of curiosity, how slow?  Any numbers you can share

Reply:
It's around 8-9x slower, over a million such calls it adds up